domains/it.json

```diff
@@ -0,0 +1,20 @@
+{
+        "owner": {
+           "username": "tr1xem",
+           "email": "kmtechnical2006@gmail.com",
+           "discord": "885063317079592961"
+        },
+    
+        "record": {
+             "CNAME": "superuser-l.github.io",
+             "TXT": [
```

Comment:
txt record can only be a string or array of string

Reply:
But it is a github pages record? how do i fix it any solutions?

Reply:
GitHub Pages use CNAME not TXT

Reply:
Using it for whole account not a single repo